src/NServiceBus.Core.Tests/Persistence/PersistenceStorageMergerTests.cs

```diff
@@ -68,4 +54,29 @@ public FakePersistence()
             }
         }
     }
+
+    [TestFixture]
+    public class When_explicitly_enabling_selected_storage
+    {
+        [Test]
+        public void Should_not_use_other_supported_storages()
+        {
+            var config = new EndpointConfiguration("MyEndpoint");
+            config.UsePersistence<FakePersistence, StorageType.Sagas>();
+            var persistences = config.Settings.Get<List<EnabledPersistence>>("PersistenceDefinitions");
+
+            var resultedEnabledPersistences = PersistenceStorageMerger.Merge(persistences, config.Settings);
+
+            Assert.IsTrue(!resultedEnabledPersistences.Any(p => p.SelectedStorages.Contains(typeof(StorageType.Subscriptions))));
```

Comment:
```suggestion
            Assert.IsFalse(resultedEnabledPersistences.Any(p => p.SelectedStorages.Contains(typeof(StorageType.Subscriptions))));
```

would this be easier to read?